2018-07-09-team-sectumsempra.md

avneetreen opened the pull request:
Team Sectumsempra blog.
RGSoC 2018

avneetreen committed:
Create Hola, Amigos.md

v-thakkar commented on blog/_posts/Hola, Amigos.md:
```diff
@@ -0,0 +1,118 @@
+# "We solemnly swear we’re up to no good."
+
```
Hi @avneet14027  ! Thanks for the PR. Please check the other blog posts for the proper markdown format. We need the whole set of parameters at the start.

v-thakkar commented on blog/_posts/Hola, Amigos.md:
```diff
@@ -0,0 +1,118 @@
+
+#### Looking forward to an funfilled summer,
+#### Team Sectumsempra
+
```
No need of extra blank lines. Please remove them.

v-thakkar commented on blog/_posts/Hola, Amigos.md:
```diff
@@ -0,0 +1,118 @@
+ Footprints, is what we wish to leave behind. :D 
+ 
+ 
+ ![Alt Text](https://github.com/avneet14027/Miscellaneous/blob/master/WhatsApp%20Image%202018-07-09%20at%205.13.08%20PM.jpeg)
```
Add image description and image-credits class for adding image credits. Same for the GIF at the bottom.

v-thakkar commented on blog/_posts/Hola, Amigos.md:
```diff
@@ -0,0 +1,118 @@
+# "We solemnly swear we’re up to no good."
+
+## How delayed rewards brought us extra happiness!
+After a thorough application process, a long wait and an interview, we thought we might just be able to make it through, and were eagerly waiting for that one “Welcome to RGSoC” email. In time, an email popped up in our feed which, lured us into thinking we emerged victorious,  but Alas! When the first round of scholars was announced, we were heart-broken that we didn't make the cut. However, the ...
```
Change victorious -> victoriously.

v-thakkar commented on blog/_posts/Hola, Amigos.md:
```diff
@@ -0,0 +1,118 @@
+
+Elevating Medical Sciences using computers and technology is my passion, and it will be my future. I want to become a Computer Scientist and be known for my talent, and not because I’m a *woman*. I want people to understand that *I am more than what they can see*."
+
+I have had this habit of imaginng things in my head, things that I aspire to do and achieve, but the results are not always the way I expected them to be. But then, I take a deep breath and reflect back, and am reminded of the famous quote from the *Harry Potter series*
```
Fix the typo here: 'imagining'.

v-thakkar commented on blog/_posts/Hola, Amigos.md:
```diff
@@ -0,0 +1,118 @@
+### Our Official coaches:
+We are grateful to them for agreeing to coach us, guide us, and help us grow and learn, as we embark upon this journey. Again, we are extremely thankful to him for all the mentoring at the time of applications.
+
+ - [**Louis Jenkins**](https://louisjenkinscs.github.io/): He is a  recent Computer Science graduate from Bloomsburg interests and experience in the low-level areas such as compilers, operating systems, and the design and implementation of high-performance data structures. [Source](https://louisjenkinscs.github.io/)
```
Not sure why you have added the same website/linkedin links twice. Please remove the links with 'Source' texts.

v-thakkar commented on blog/_posts/Hola, Amigos.md:
```diff
@@ -0,0 +1,118 @@
+
+- [**Kushal Singh**](https://in.linkedin.com/in/kushalsingh007): He is currently working as a platform engineer at Direct-I, India, and has previously been part of Google Summer Of Code. [Source](https://in.linkedin.com/in/kushalsingh007)
+
+- [**Lydia Duncan**](https://github.com/lydia-duncan)
```
It looks weird to have small introduction text for everyone except this person. Please either add for all or none.

avneetreen committed:
Update Hola, Amigos.md

avneetreen commented:
Hi Vaishali,
I have made the changes as suggested by you. 
Thanks,
Avneet

v-thakkar commented on blog/_posts/Hola, Amigos.md:
```diff
@@ -0,0 +1,97 @@
+---
+**title:** We solemnly swear we’re upto no good!
```
Hi @avneet14027 ! You would need to use exact format as used in other files for declaring parameters. For example, no use of '**', permalink same as file name etc. Please check and test them locally.

v-thakkar commented on blog/_posts/Hola, Amigos.md:
```diff
@@ -0,0 +1,97 @@
+>                    - H.W. Longfellow 
+ Footprints, is what we wish to leave behind. :D 
+ 
+ ![Alt Text](https://github.com/avneet14027/images/blob/master/2018-07-09-team-sectumsempra-blog-teampic.jpeg)
```
Please add image description as well. For example, who is in the image etc. And then in brackets, you can write about image source. As the blog post is from a team and not a single person, 'self' is a confusing word as a source. You need to specify the name or device etc.

avneetreen committed:
Update and rename Hola, Amigos.md to 2018-07-09-team-sectumsempra.md

avneetreen committed:
Update 2018-07-09-team-sectumsempra.md

avneetreen committed:
Update 2018-07-09-team-sectumsempra.md

avneetreen committed:
Update 2018-07-09-team-sectumsempra.md

avneetreen commented:
Hey @nerdyvaishali , I have updated the file as per the changes requested!
Thanks,
Avneet

Commit:
Team Sectumsempra blog post edditing

Changed the date of the blog post
Added the images to the blog folder
Minor formatting changes and typo correction
